app/src/main/java/io/gnosis/safe/ui/settings/app/GetInTouchFragment.kt

```diff
@@ -86,7 +85,12 @@ class GetInTouchFragment : BaseViewBindingFragment<FragmentGetInTouchBinding>()
             val feedback = helper.createFeedbackText(requireContext())
             val intent = Intent(Intent.ACTION_SENDTO).apply {
                 type = "text/html"
-                data = Uri.parse("mailto:${getString(R.string.email_feedback)}")
+                // gmail is ignoring intent extras
+                data = Uri.parse(
+                    "mailto:${getString(R.string.email_feedback)}" +
+                            "?subject=${getString(R.string.feedback_subject)}" +
```

Comment:
Does this work with other email clients as well?

Reply:
yes